src/commands/serviceConnector/IServiceConnectorContext.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { ICreateLinkerContext } from "@microsoft/vscode-azext-serviceconnector";
+import { ExecuteActivityContext } from "@microsoft/vscode-azext-utils";
+import { ContainerAppModel } from "../../tree/ContainerAppItem";
+
+export interface IServiceConnectorContext extends ICreateLinkerContext {
+    containerApp: ContainerAppModel;
```

Comment:
> I extended from the ICreateLinkerContext since I needed the scope.

Does it not make sense to extend from both?

> Yeah, `IStreamLogsContext` and `IAddScaleRuleWizardContext` could as well though `IContainerAppContext` has `containerApp` as an optional property. Not sure if that gunks things up a bit.

Yeah, there are a few contexts that still need to be migrated to extend from `IContainerAppContext` since it was only recently added.  Scale rules is definitely one of those contexts.